bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -263,6 +283,55 @@ const BaseContainer = withTracker(() => {
     ...subscriptionErrorHandler,
   });
 
+  Breakouts.find().observeChanges({
+    added() {
+      breakoutNotified = false;
+    },
+    removed() {
+      if (!AudioService.isUsingAudio() && !breakoutNotified) {
+        if (meeting && !meeting.meetingEnded) {
+          notify(
+            <FormattedMessage
+              id="app.toast.breakoutRoomEnded"
+              description="message when the breakout room is ended"
+            />,
+            'info',
+            'rooms',
+          );
+        }
+        breakoutNotified = true;
+      }
+    },
+  });
+
+  Meetings.find({ meetingId }).observe({
+    changed: (newDocument, oldDocument) => {
```

Comment:
This could be a huge performance killer :thinking: 
Why do we need to know whether the recording status changed in Base?

Reply:
https://github.com/bigbluebutton/bigbluebutton/pull/7074#issuecomment-481379556
This code was causing errors where it was being called because it tried to perform an update after unmount